lib/schemaUtils.js

```diff
@@ -1906,12 +1906,14 @@ module.exports = {
     // Add collection variables to the variableStore.
     sanitizeResult.collectionVars.forEach((element) => {
       if (!variableStore[element.name]) {
-        variableStore[element.name] = {
-          id: element.name,
-          value: element.default || '',
-          description: element.description,
-          type: 'collection'
-        };
+        let fakedData = options.schemaFaker ?
+            safeSchemaFaker(element.schema || {}, options.requestParametersResolution, PROCESSING_TYPE.CONVERSION,
+              PARAMETER_SOURCE.REQUEST, components, SCHEMA_FORMATS.DEFAULT, options.indentCharacter, schemaCache,
+              options.stackLimit) : '',
+          convertedPathVar = _.get(this.convertParamsWithStyle(element, fakedData, PARAMETER_SOURCE.REQUEST,
+            components, schemaCache), '[0]', {});
+
+        variableStore[element.name] = _.assign(convertedPathVar, { id: element.name, type: 'collection' });
```

Comment:
@VShingala 
Should we also respect the default value if provided in schema?

Reply:
@umeshp7 That will be done by `json-schema-faker` based on existing logic. This is same as what we are doing for normal path variables value.